test/sqs.lua

```diff
@@ -14,13 +14,22 @@ dump_res('create', r)
 
 local QueueUrl = r.value.CreateQueueResponse.value.CreateQueueResult.value.QueueUrl.value
 print("QueueUrl:", QueueUrl)
+local jsonmsg = [[{"email_id":2,"contact_id":"1-xxxmlxx@hotmail.es","bulk_id":1,"email_version_id":14,"client_id":1}]]
```

Comment:
Most changes in this file were from the sync with upstream.  My additional change was the "1-xxxmlxx@hotmail.es" to include xml in the SQS payload.  Without the xml decoding fix this will cause the Body.value receiveMessage() response to be nil.